port/ztpf/omrosdump_helpers.c

```diff
@@ -932,7 +932,7 @@ buildELFNoteArea(Elf64_Nhdr *buffer, DIB *dibPtr)
 	nsh_lastbr->n_type = 0x500; /* Type = 500                */
 	memcpy(nsh_lastbr + 1, "ZTPF", 5); /* Set "ZTPF" name ...        */
 	wDest += 0x14; /*  and point at data        */
-	sprintf(wDest, "SE-%4.4d OPR-%c%6.6X PGM-%4.4s %5.5c %8.8s", eriPtr->eriseq,
+	sprintf((const char *)wDest, "SE-%4.4d OPR-%c%6.6X PGM-%4.4s %5.5c %8.8s", eriPtr->eriseq,

```

Comment:
sprintf writes to the buffer addressed by the first argument: `const` is inappropriate here.

Reply:
@keithc61 Thanks for taking a look.  Updates made.  
New Internal Build id=364104.